### DISCUSSION TITLE
CloudLogCat in Linux

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#1919] [Q&A]
Hello!

I am trying to connect FLRig to CloudLog using CloudLogCat, which I have downloaded and compiled. How to do I run it and access the CloudLogCat GUI so I can put in the correct settings? I am very new to Linux and cannot figure out how to RUN CloudLogCat. Which file do I double click on, or what do I need to do in Terminal to run it?

Apologies for being such a newbie.

Thanks
Scott, W1AL

**Comment by magicbug** [comment]
Think @phl0 will be best chance of an answer on this don't use FLRig here!

**Comment by phl0** [comment]
Well I run flrig to feed the CAT data into CloudLogCatQt and CloudLogCatQt feeds the CAT data received from flrig to Cloudlog. So the question is how to run CloudLogCatQt? Did you compile according to its README?

**Comment by soakland5** [comment]
Yes I can set it all up but am new to Linux and don’t know how to run the
program CloudLogCat. Yes I compiled it according to the readme. I have a
folder all set and it has files but don’t know which one to run or how.

Thanks
Scott

**Reply by phl0** [reply]
Ok, I guess you compiled the stuff on the console. Then just execute the binary from there using:

```
   $ ./CloudLogCatQt
```

**Reply by phl0** [reply]
Without the trailing dollar sign (just a designator for the command prompt).

**Comment by soakland5** [comment]
Sorry I included the wrong photo

**Comment by soakland5** [comment]
I tried but no good. Like I said I’m new to Linux. What am I doing wrong ?

**Reply by phl0** [reply]
Can you paste the output or show a screenshot of the result pls?

**Comment by soakland5** [comment]
Sure here it is. It keeps saying "is a directory".

***@***.***:~$ $ ./CloudLogCatQt
$: command not found
***@***.***:~$ ./CloudLogCatQt
bash: ./CloudLogCatQt: Is a directory
***@***.***:~$ cd CloudLogCatQt
***@***.***:~/CloudLogCatQt$ ./CloudLogCatQt
bash: ./CloudLogCatQt: Is a directory
***@***.***:~/CloudLogCatQt$ $ ./CloudLogCatQt
$: command not found
***@***.***:~/CloudLogCatQt$ $./CloudLogCatQt
bash: $./CloudLogCatQt: No such file or directory
***@***.***:~/CloudLogCatQt$ ./CloudLogCatQt
bash: ./CloudLogCatQt: Is a directory
***@***.***:~/CloudLogCatQt$

**Reply by phl0** [reply]
You should change to the buiöd directory before with

```
$ cd build
$ ./CloudLogCatQt
```

As I said enter without the leading dollar signs.

**Comment by soakland5** [comment]
Hi Peter:

Thanks. I'm going back to Windows now and wish to interface Gridtracker
with Cloudlog but could not get it to connect. Can you give me correct URL?
It keeps failing to connect. I tried 127.0.0.1 but it tells me it's a bad
URL. I creatyed an API key and pasted that in to the API box and put w1al
as my station profile ID. I even tried the Gridtracker site and thatg info
didn't work. IDeas?

Thanks
Scott

**Comment by phl0** [comment]
I can only offer again to support via remote desktop and explain. This is not too hard to set up. Though might a little difficult as Linux newbie.

**Comment by soakland5** [comment]
Thanks but Linux is too much for me. Went back to Windows even though I
hate its terrible security vulnerabilities and crazy quirks. I found Hamlog
online from Russia that is browser based and seems promising. They use a
windows tray app to feed UDP  log entries.

I still have Linux on a 5 year old I-Mac and it’s pretty fast. Just not
suitable for ham non programmers lol.

Thanks for your help!

Scott
